tests/params/test_freestream.py

```diff
@@ -40,10 +40,12 @@ def test_freesteam():
         to_file_from_file_test(fs)
 
         p = fl.Flow360Params(
+            geometry=fl.Geometry(),
```

Comment:
I don't like requesting user to specify empty structure. Let's handle it differently to let user omitting this parameter.

Reply:
Removed and added the empty as default